source/Nevermore.Tests/Util/DataModificationQueryBuilderFixture.UpdateWithManyRelatedDocuments.approved.txt

```diff
@@ -1,24 +1,18 @@
 UPDATE [dbo].[TestDocumentTbl]  SET [AColumn] = @AColumn, [JSON] = @JSON WHERE [Id] = @Id
 
 DECLARE @references as TABLE (Reference nvarchar(400), ReferenceTable nvarchar(400))
-
-DELETE FROM @references
```

Comment:
Why don't we need this `DELETE` statement anymore? was it doing nothing before?

Reply:
Yep, did nothing as there weren't any records in the table variable at this point.